docs/usage/vmware-netplan-guestinfo-metadata.md

```diff
@@ -0,0 +1,36 @@
+---
+nav_order: 2
+parent: Usage
+---
+
+# VMware Netplan guestinfo metadata
+
+The network environment can vary between VMware servers and instead of leaking these requirements into userdata snippets, a well-known guestinfo metadata field can be used.
+The guestinfo metadata field is OS-independent and supported by cloud-init and afterburn. It requires Netplan to render the Netplan YAML format to either NetworkManager or systemd-networkd configuration files.
+By default, systemd-networkd units are used. Since the renderer backend is defined in the Netplan config itself, requiring NetworkManager would rule out support for systems that don't use it (unless they would postprocess the `renderer` field to force it to `networkd`). Since systemd-networkd can work in parallel with NetworkManager, it's expected that the renderer field is left to its default.
+
+The afterburn invocation is as follows, where `FOLDER` could be `/run/netplan/`:
+
+```
+afterburn multi --netplan-configs FOLDER --provider vmware
+```
+
+Afterwards, `netplan generate` can be used to render the config files. If that is done before `systemd-networkd` runs, this is enough, but if the network already is up, `netplan apply` should be used instead.
```

Comment:
This type of stuff belongs better in `distro.md` since it's not user-facing but for integrators.

Reply:
Moved, thanks